test/e2e/create_cluster_test.go

```diff
@@ -129,7 +129,6 @@ func TestCreateClusterRequestServingIsolation(t *testing.T) {
 
 	e2eutil.NewHypershiftTest(t, ctx, func(t *testing.T, g Gomega, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
 		guestClient := e2eutil.WaitForGuestClient(t, testContext, mgtClient, hostedCluster)
-		e2eutil.EnsurePSANotPrivileged(t, ctx, guestClient)
```

Comment:
Per our slack discussion, I think you need to first remove the test in a separate PR in order for PR testing to pass on this PR.

Reply:
https://github.com/openshift/hypershift/pull/4855 will hopefully resolve this.